Backend/Remora.Discord.API/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -668,6 +668,18 @@ private static JsonSerializerOptions AddGuildObjectConverters(this JsonSerialize
         options.AddDataObjectConverter<IWelcomeScreen, WelcomeScreen>();
         options.AddDataObjectConverter<IWelcomeScreenChannel, WelcomeScreenChannel>();
         options.AddDataObjectConverter<IGuildThreadQueryResponse, GuildThreadQueryResponse>();
+        options.AddDataObjectConverter<IGuildOnboarding, GuildOnboarding>()
+            .WithPropertyName(o => o.DefaultChannelIDs, "default_channel_ids")
```

Comment:
This override should not be required.

Reply:
I tested without it and tests started to fail. Therefore it is required. I suppose its cause of the "...IDs" part.

Reply:
Interesting - will look into this later.

Reply:
Must be the SnakeCaseNamingPolicy that is making things up.